x-pack/plugins/cases/public/components/custom_fields/toggle/create.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { UseField } from '@kbn/es-ui-shared-plugin/static/forms/hook_form_lib';
+import { ToggleField } from '@kbn/es-ui-shared-plugin/static/forms/components';
+import type { CustomFieldType } from '../types';
+
+const CreateComponent: CustomFieldType['Create'] = ({ customFieldConfiguration, isLoading }) => {
+  const { key, label, type } = customFieldConfiguration;
+
+  return (
+    <UseField
+      path={`customFields.${key}`}
+      component={ToggleField}
+      defaultValue={false}
+      type={type}
```

Comment:
`type` is not needed.

Reply:
removed in https://github.com/elastic/kibana/pull/166969/commits/8e78b121501bef04ce2c2789936e0a3cc912b46d